src/tools/pub/glossary.md

```diff
@@ -146,16 +146,34 @@ and C are transitive ones.
 
 ## Uploader
```

Comment:
Maybe indent this subsection and the verified publisher subsection below under a new overall section?

```
## Managing who has rights to publish

### Uploader

### Verified publisher
```

Reply:
This is a glossary, not a task oriented doc, so alpha order is the rule.